### PR TITLE
[WIP] Feature/stjs 625 destroy spa

### PR DESCRIPTION
**Pull request by wirus15**

### CODE REVIEW

File: src/application/core/shared/EventTypes.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  SUBMIT_FORM: 'SUBMIT_FORM',
+  THREEDINIT_REQUEST: 'THREEDINIT_REQUEST',
+  THREEDINIT_RESPONSE: 'THREEDINIT_RESPONSE',
+  THREEDQUERY: 'THREEDQUERY',
+  TRANSACTION_COMPLETE: 'TRANSACTION_COMPLETE',
+  UPDATE_JWT: 'UPDATE_JWT',
+  UPDATE_MERCHANT_FIELDS: 'UPDATE_MERCHANT_FIELDS',
+  SUBSCRIBE: 'SUBSCRIBE',
+  CONFIG_CHECK: 'ST_CONFIG_CHECK'
+};
```
Comment (pgs-gmartowski) [inline src/application/core/shared/EventTypes.ts:50]: 👍

File: src/client/ST.ts [hunk 2]
```diff
@@ -190,7 +189,7 @@ class ST {
     this._communicator.close();
   }
 
-  private init(): void {
+  public init(): void {
     // TODO theres probably a better way rather than having to remember to update Selectors
     Selectors.MERCHANT_FORM_SELECTOR = this._config.formId;
```
Comment (pgs-gmartowski) [inline src/client/ST.ts:194]: info: I think this is not needed anymore - now we have default values set in `ConfigResolver`

Comment (wirus15) [reply src/client/ST.ts:194]: It depends whether the app uses the value from the config or from the `Selectors`. Nevertheless this is outside the scope of this task :)

**Comment by pgs-gmartowski** [comment]
👍

**Pull request merged**